perception/compare_map_segmentation/README.md

```diff
@@ -75,13 +75,14 @@ WIP
 
 #### Parameters
 
-| Name                            | Type  | Description                                                                                  | Default value |
-| :------------------------------ | :---- | :------------------------------------------------------------------------------------------- | :------------ |
-| `use_dynamic_map_loading`       | bool  | map loading mode selection, `true` for dynamic map loading, `false` for static map loading   | true          |
-| `distance_threshold`            | float | VoxelGrid's leaf_size also the threshold to check distance between input point and map point | 0.5           |
-| `map_update_distance_threshold` | float | Threshold of vehicle movement distance when map update is necessary [m]                      | 10.0          |
-| `map_loader_radius`             | float | Radius of map need to be loaded (in dynamic map loading) [m]                                 | 150.0         |
-| `timer_interval_ms`             | int   | time interval of timer to check if update the map is necessary (in dynamic map loading) [ms] | 100           |
+| Name                            | Type  | Description                                                                                                                             | Default value |
```

Comment:
This section is for "Other Filters".
If these parameters are only for voxel_based_compare_map, it should be separated

Reply:
@miursh Here "Other Filters" mean other than elevation map filter. 

> If these parameters are only for voxel_based_compare_map, it should be separated

Yes,Part of them for dynamic map loading currently only support for voxel_based_compare_map, other compare map filters still ignore them since no dynamic map loader is available yet. They are will be used for other compare map filter in dynamic map loading mode  that I'm working on here https://github.com/autowarefoundation/autoware.universe/pull/3365. 
Is it possible to keep them or should I separate with the remaining compare map filters?

Reply:
If all of them will use the same params, it's ok to keep this